### DISCUSSION TITLE
DS920+ frequently  got link down and up

### DISCUSSION THREAD

**Discussion by iandyilove** [bb-qq/r8152#350] [Q&A]
System version:DSM 7.2.1-69057 Update 1 .
I use 8152 chip usb3 2.5gbps adapter.I installed the driver into m2 nvme (r8152-geminilake-2.17.1-1_7.2.spk)
![screenshot-20231108-095858](https://github.com/bb-qq/r8152/assets/11225098/7d5d83a9-912d-40b4-869b-07ef20a2d0c8)
The network frequently  got link down and up,and the network speed drop down to 100Mbps
I'm so confused.Why?Thx.

**Comment by bb-qq** [comment]
Please try to disable Energy Efficient Ethernet(EEE).

**Reply by TDS1995** [reply]
Hello, I am struggling with the same problem.
However, I cannot find the option to deactivate Energy Efficient Ethernet (EEE).
Can anyone tell me how to do this?

Thank you very much!

**Reply by bb-qq** [reply]
You can disable EEE of r8152 with the following command:
```
ethtool --set-eee eth2 eee off
```
However, I recommend disabling it on the opposite Ethernet switch or router side.
